lib/workers/pr/index.js

```diff
@@ -76,6 +77,12 @@ async function ensurePr(prConfig) {
       return null;
     }
     logger.debug('Branch status success');
+  } else if (
+    config.prCreation === 'approval' &&
+    !existingPr &&
+    masterIssueCheck !== 'approvePr'
+  ) {
+    return 'needs-approval';
```

Comment:
Let's make this 'needs-pr-approval' instead of 'needs-approval' from the very start so that we don't have to remap it.